department-of-veterans-affairs/va.gov-cms
# VACMS-10774: VACMS-10904: Adds help text with kb link on paths fields; changes 'add another item' button text

## Description

Closes #10774
Closes #10904

## Screenshots
![Screen Shot 2022-12-07 at 11 35 45 AM](https://user-images.githubusercontent.com/221539/206278910-2560d5b8-85a1-4bfd-951d-2f9f07463207.png)

## QA steps

As a content_admin
1. Visit the add 'Promo Banners' node creation link: [/node/add/promo_banner](/node/add/promo_banner)
   - [x] (For #10774) Validate that the 'Paths' field help text is: `Add path(s) for pages that should display this content. All paths must begin with / and may use wildcards (*) or exclusions (!). (<a href="/help/homepage-alerts-and-promo-banners/promo-banners" target=“_blank”>See further guidelines</a>)`
   - [x] (For #10904) Validate the button displays 'Add another path' instead of 'Add another item'.
2. Visit the add 'Full Width Alert' node creation link: [/node/add/banner](/node/add/banner)
   - [x] (For #10774) Validate that the 'Paths' field help text is: `Add path(s) for pages that should display this content. All paths must begin with / and may use wildcards (*) or exclusions (!). (<a href="/help/homepage-alerts-and-promo-banners/promo-banners" target=“_blank”>See further guidelines</a>)`
   - [x] (For #10904) Validate the button displays 'Add another path' instead of 'Add another item'.



### Definition of Done


### Select Team for PR review
